src/Core/Services/PclCryptoFunctionService.cs

```diff
@@ -49,18 +49,15 @@ public Task<byte[]> Argon2Async(string password, string salt, int iterations, in
             password = NormalizePassword(password);
```

Comment:
revert changes to this file

Reply:
Done in 8aabd1c4f328095572b66021ed12e54a63378a5b